Dockerfile.resotobase

```diff
@@ -80,50 +80,51 @@ RUN mkdir -p /build-python
-RUN . /usr/local/resoto-venv-python3/bin/activate && python -m pip wheel -w /build-python -f /build-python .
-RUN . /usr/local/resoto-venv-pypy3/bin/activate && pypy3 -m pip wheel -w /build-pypy -f /build-pypy .
+RUN . /usr/local/resoto-venv-python3/bin/activate && python -m poetry install
+RUN . /usr/local/resoto-venv-pypy3/bin/activate && pypy3 -m poetry install
 
 # Build resotocore
 COPY resotocore /usr/src/resotocore
 WORKDIR /usr/src/resotocore
 #RUN if [ "X${TESTS:-false}" = Xtrue ]; then nohup bash -c "/usr/local/db/bin/arangod --database.directory /tmp --server.endpoint tcp://127.0.0.1:8529 --database.password root &"; sleep 5; tox; fi
-RUN . /usr/local/resoto-venv-python3/bin/activate && python -m pip wheel -w /build-python -f /build-python .
-RUN . /usr/local/resoto-venv-pypy3/bin/activate && pypy3 -m pip wheel -w /build-pypy -f /build-pypy .
+RUN . /usr/local/resoto-venv-python3/bin/activate && python -m poetry install
+RUN . /usr/local/resoto-venv-pypy3/bin/activate && pypy3 -m poetry install
 
 # Build resotoworker
 COPY resotoworker /usr/src/resotoworker
 WORKDIR /usr/src/resotoworker
 RUN if [ "X${TESTS:-false}" = Xtrue ]; then . /usr/local/resoto-venv-python3/bin/activate && tox; fi
-RUN . /usr/local/resoto-venv-python3/bin/activate && python -m pip wheel -w /build-python -f /build-python .
+RUN . /usr/local/resoto-venv-python3/bin/activate && python -m poetry install
 
 # Build resotometrics
 COPY resotometrics /usr/src/resotometrics
 WORKDIR /usr/src/resotometrics
 RUN if [ "X${TESTS:-false}" = Xtrue ]; then . /usr/local/resoto-venv-python3/bin/activate && tox; fi
-RUN . /usr/local/resoto-venv-python3/bin/activate && python -m pip wheel -w /build-python -f /build-python .
+RUN . /usr/local/resoto-venv-python3/bin/activate && python -m poetry install
 
 # Build resotoshell
 COPY resotoshell /usr/src/resotoshell
 WORKDIR /usr/src/resotoshell
 RUN if [ "X${TESTS:-false}" = Xtrue ]; then . /usr/local/resoto-venv-python3/bin/activate && tox; fi
-RUN . /usr/local/resoto-venv-python3/bin/activate && python -m pip wheel -w /build-python -f /build-python .
+RUN . /usr/local/resoto-venv-python3/bin/activate && python -m poetry install
 
 # Build resoto plugins
 COPY plugins /usr/src/plugins
 WORKDIR /usr/src
 RUN if [ "X${TESTS:-false}" = Xtrue ]; then . /usr/local/resoto-venv-python3/bin/activate && find plugins/ -name tox.ini | while read toxini; do cd $(dirname "$toxini") && tox && cd - || exit 1; done; fi
-RUN . /usr/local/resoto-venv-python3/bin/activate && find plugins/ -maxdepth 1 -mindepth 1 -type d -print0 | xargs -0 python -m pip wheel -w /build-python -f /build-python
-
-# Install all wheels
-RUN . /usr/local/resoto-venv-python3/bin/activate && python -m pip install -f /build-python /build-python/*.whl
-RUN . /usr/local/resoto-venv-pypy3/bin/activate && pypy3 -m pip install -f /build-pypy /build-pypy/*.whl
+RUN for plugin_dir in $(find plugins/ -maxdepth 1 -mindepth 1 -type d); do \
```

Comment:
A for loop won't work here because some of the plugins depend on another. E.g. the cleanup_aws_* depend on aws. If it works it's only through luck, because it got to the dependent plugin first.

Reply:
good point. The logic here is the same as it was before when the output of the `find` command was used. I will create a ticket to rewrite this part.